subiquitycore/snapd.py

```diff
@@ -41,13 +41,15 @@ def get(self, path, **args):
         if args:
             path += "?" + urlencode(args)
         with requests_unixsocket.Session() as session:
-            return session.get(self.url_base + path, timeout=60)
+            return session.get(self.url_base + path, timeout=600)
```

Comment:
Please consider adding a comment linking to the LP bug.

Reply:
I like that it's present in the commit message though, so feel free to disregard if you think it's enoug.

Reply:
Yeah I'd like a comment near the code too please